data/pigui/views/mainmenu.lua

```diff
@@ -197,7 +196,7 @@ local function showMainMenu()
 	ui.setNextWindowPos(Vector2(110,65),'Always')
 	ui.withStyleColors({["WindowBg"]=colors.transparent}, function()
 		ui.window("headingWindow", overlayWindowFlags, function()
-			ui.withFont("orbiteer",36 * (ui.screenHeight/1200),function() ui.text("Pioneer") end)
+			ui.withFont("orbiteer", ui.rescaleUI(36), function() ui.text("Pioneer") end)
```

Comment:
I'd recommend these calls in the main menu use one of the actual `ui.fonts.orbiteer` font objects instead of manual scaling every frame.

Reply:
"Done!" :smile: